requirements.txt

```diff
@@ -0,0 +1,7 @@
+xlrd==0.9.3
+unicodecsv
+jinja2
+beautifulsoup4
+datapackage
+goodtables
```

Comment:
Do we really use these dependencies? Eg jinja2 or Unicode CSV. datapackage and goodtables should be included in dataflows

Reply:
Good point. Removed those